### PR TITLE
Add Road Worker profession and a safety vest.

### PR DESCRIPTION
**Pull request by Test-User9182**
#### Summary

Content "Add Road Worker profession and a safety vest."

#### Purpose of change

Adds a Road Worker start, so one can start with a road roller.
Adds safety vests because all road workers would be wearing one (when working), can also be used in other technician-related places (zombie technician, power substations, etc.).

#### Testing

Valid json. Works as intended when using it when creating a new player.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -4149,5 +4149,33 @@
...
+          "technician_shirt_gray",
+          "tool_belt",
+          "boots"
+        ],
+        "entries": [ { "item": "shovel", "custom-flags": "auto_wield" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    },
+    "traits": "PSYCHOPATH"
```
Comment (Night-Pryanik) [inline data/json/professions.json:4179]: Is this trait really needed for this profession?

Comment (Test-User9182) [reply data/json/professions.json:4179]: Not really needed as the name goes, but it fits the description and the rest quite well

Comment (kevingranade) [inline data/json/professions.json:4179]:
```suggestion
    }
```

**Comment by kevingranade** [comment]
Starting with a road roller is a no-go.

**Pull request closed without merging**